src/shared/components/mutationTypePanel/LollipopMutationTypePanel.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from "react";
+
+import {Mutation} from "../../api/generated/CBioPortalAPI";
+import {IMobXApplicationDataStore} from "../../lib/IMobXApplicationDataStore";
+import {observer} from "mobx-react";
+import {computed} from "mobx";
+import MutationTypePanel from "./MutationTypePanel";
+import {getLollipopMutationType, LollipopMutationType} from "../../lib/getCanonicalMutationType";
+import {MutationTypePanelButton} from "./MutationTypePanel";
+
+type LollipopMutationTypePanelProps = {
+    dataStore:IMobXApplicationDataStore<Mutation[]>;
+    missenseColor:string;
+    inframeColor:string;
+    truncatingColor:string;
+    otherColor:string;
+};
+
+const buttonOrder:LollipopMutationType[] = ["missense", "truncating", "inframe", "other"];
+
+@observer
+export default class LollipopMutationTypePanel extends React.Component<LollipopMutationTypePanelProps, {}> {
```

Comment:
`Lollipop` does not feel right here. Because there are no lollipops in this panel.

Can we name this one to `MutationTypePanel` and the lower level component to something generic like `TypePanel`, `CountPanel`, `TypeCountPanel`, or something better?